executor/runtime/docker/capabilities.go

```diff
@@ -54,6 +55,10 @@ func setupAdditionalCapabilities(c runtimeTypes.Container, hostCfg *container.Ho
 			PathInContainer:   fuseDev,
 			CgroupPermissions: "rmw",
 		})
+
+		if apparmorProfile == defaultApparmorProfile {
```

Comment:
Is this if redundant?

Reply:
If `c.AppArmorProfile()` is non-nil ([L44](https://github.com/Netflix/titus-executor/pull/607/files#diff-eda016907a75ec19335cdc60f120452ec1f19a7ea4d5b717d0699b438be3936eR44)), then `apparmorProfile` will be set to something that's not the default.  I only want to set it back to `docker_fuse` if `apparmorProfile` is nil - in other words, the user didn't explicitly choose to override the profile.

Reply:
But if someone does manually set `docker_titus` (a non nil setting), this code will *still* override to `docker_fuse`.
I think that is OK though, this will be such a rare thing.
Heck if the logic is here in the executor, I dare say we shouldn't allow users to set AA profiles in the first place?

Reply:
We don't allow setting AA profiles via the API.  I considered ripping this code out altogether, then realized that I would probably want to use this functionality when iterating on AA changes with `titus-standalone` (which is currently the only way you could set this annotation, other than creating a pod in apiserver directly).